### PR TITLE
Add datafile from Nextcloud

### PR DESCRIPTION
**Pull request by lokal-profil**

### CODE REVIEW

File: python_scripts/AddressPoint.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python2.7
```
Comment (sebastian-berlin-wmse) [inline python_scripts/AddressPoint.py:1]: Why not Python 3?

File: python_scripts/AddressPoint.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+# -*- coding: utf-8 -*-
+#
+#Short script for formating AddressPoint datafile
```
Comment (sebastian-berlin-wmse) [inline python_scripts/AddressPoint.py]: Typo: formatting

File: python_scripts/AddressPoint.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+#run using python AddressPoint.py -i <filnamn>
+#
+import codecs, sys, getopt
```
Comment (sebastian-berlin-wmse) [inline python_scripts/AddressPoint.py]: One import per line: https://www.python.org/dev/peps/pep-0008/#imports.

File: python_scripts/AddressPoint.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+def main(argv):
+    inputfile = ''
+    outputfile = ''
+    try:
+        opts, args = getopt.getopt(argv,"hi:")
```
Comment (sebastian-berlin-wmse) [inline python_scripts/AddressPoint.py]: Space after comma (in call). More cases below.

File: python_scripts/AddressPoint.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        print u'python AddressPoint.py -i <filnamn>'
+        sys.exit(2)
+    for opt, arg in opts:
+        if opt == '-h':
+            print u'python AddressPoint.py -i <filnamn>'
+            sys.exit()
+        elif opt == "-i":
+            inputfile = arg
+    if len(inputfile) > 0:
+        outputfile = u'%s-mod.txt' %inputfile[:-4]
```
Comment (sebastian-berlin-wmse) [inline python_scripts/AddressPoint.py]: The last expression removes file extension, right? Use [os.path.splitext](https://docs.python.org/3.5/library/os.path.html#os.path.splitext) for robustness and readability.

File: python_scripts/AddressPoint.py (hunk 4 above)
Comment (sebastian-berlin-wmse) [reply python_scripts/AddressPoint.py]: Some spaces are also missing around equal signs.

File: python_scripts/AddressPoint.py [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        try:
+            fIn=codecs.open(inputfile,'r','Latin-1')
+        except IOError:
+            print u'Skrev du verkligen rätt filnamn?'
+            sys.exit(2)
+        lines = fIn.read().split(u'\n')
+        fIn.close()
+        
+        fOut=codecs.open(outputfile,'w','utf8')
+        for l in lines:
```
Comment (sebastian-berlin-wmse) [inline python_scripts/AddressPoint.py]: PEP 8 recommends [not using `l` as variable name](https://www.python.org/dev/peps/pep-0008/#id38).

File: python_scripts/AddressPoint.py [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+            if len(l)==0: continue
+            p = l.split(u';')
+            p[2] = p[2].title() #first name
+            p[3] = p[3].title() #last name
+            p[4] = u'%s %s'.strip() %(p[4].title(),p[5]) #To address
+            p[5] = '' #street number
+            p[12] = u'%s %s'.strip() %(p[12].title(),p[13]) #To address
+            p[13] = '' #street number
+            l= ';'.join(p)
```
Comment (sebastian-berlin-wmse) [inline python_scripts/AddressPoint.py]: Use different variable for the modified line.

**Review by sebastian-berlin-wmse — CHANGES_REQUESTED** [review]
I haven't tried running since I have no input. Some variable names could be changed to make things easier to understand/follow naming conventions.

File: python_scripts/AddressPoint.py (hunk 1 above)
Comment (lokal-profil) [reply python_scripts/AddressPoint.py:1]: This is an old file that I came across during integritetsrensing (not even sure if I wrote it or not). Honestly not sure if we still need it but moving it here made more sense than keeping it in the weirdo location I found it.

Comment (sebastian-berlin-wmse) [reply python_scripts/AddressPoint.py:1]: Ok, let's keep it for now then.

**Review by sebastian-berlin-wmse — APPROVED** [review]

**Pull request merged**